src/main/java/org/folio/edge/patron/PatronHandler.java

```diff
@@ -61,6 +62,12 @@ protected void handleCommon(RoutingContext ctx, String[] requiredParams, String[
       badRequest(ctx, "Missing required parameter: " + PARAM_PATRON_ID);
       return;
     }
+    if (null == ctx.request().getParam(PARAM_LIMIT)) {
+      ctx.request().params().add(PARAM_LIMIT, "10");
```

Comment:
if limit isn't specified we don't want to add one ourselves.  This would be a breaking compatibility change.

Reply:
changed the logic, removed this check